napari/_qt/dialogs/qt_activity_dialog.py

```diff
@@ -233,6 +233,8 @@ def get_pbar_from_prog(self, prog):
             for potential_parent in pbars:
                 if potential_parent.progress is prog:
                     return potential_parent
+            return None
+        return None
```

Comment:
Don't love all these `return None`, they feel unpythonic to me...

Reply:
In my case it few times allow me to notice that there is a branch combination that returns None and the function should return not None value. 

For example, docstring of this function do not point that the return is optional. Maybe there should be raised exception? 

It is easier to not Notice lack of return that explicit `retun None`